plugins/nv-filters/nvidia-videofx-filter.c

```diff
@@ -57,18 +63,20 @@ struct nv_greenscreen_data {
 	NvCVImage *src_img;     // src img in obs format (RGBA ?) on GPU
 	NvCVImage *BGR_src_img; // src img in BGR on GPU
 	NvCVImage *A_dst_img;   // mask img on GPU
-	NvCVImage *dst_img;     // mask texture
-	NvCVImage *stage;       // planar stage img used for transfer to texture
+	/* clang-format off */
+	NvCVImage *dst_img; // Greenscreen: alpha mask texture; blur: texture initialized from d3d11 (RGBA, chunky, u8)
+	/* clang-format on */
```

Comment:
I meant you can just use `/* clang-format off */` and `/* clang-format on */` for the entire struct, but align the comments manually. Ignore the column length limit.

Reply:
fixed